### PR TITLE
Update to VRCFT 5.2.3.0 & Update to PICO Connect 10.1

### PR DESCRIPTION
**Pull request by lonelyicer**
Solves https://github.com/regzo2/PicoStreamingAssistantFTUDP/issues/26

### CODE REVIEW

**Review by rogermiranda1000 — CHANGES_REQUESTED** [review]
The tests related with reading the old file fails:

image

Please, always consider when updating something, to keep it compatible with older versions. You can't assume all users will always use the latest.

If I were you, I'd try to parse only the file that we need (faceTrackingTransferProtocol), and ignore everything else. That way we'll avoid problems in the future, while keeping compatibility. Thanks.

File: PicoStreamingAssistantFTUDP.sln [hunk 1]
```diff
@@ -13,7 +13,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VRCFaceTracking.Core", "..\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pico4SAFTExtTrackingModule", "PicoStreamingAssistantFTUDP\Pico4SAFTExtTrackingModule.csproj", "{C5FDC7A7-FBA2-46F2-968C-FF81A57C1ED8}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PicoStreamingAssistantFTTests", "PicoStreamingAssistantFTTests\PicoStreamingAssistantFTTests.csproj", "{C4623909-EA05-45B7-B949-31ECB715EA2F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PicoStreamingAssistantFTTests", "PicoStreamingAssistantFTTests\PicoStreamingAssistantFTTests.csproj", "{C4623909-EA05-45B7-B949-31ECB715EA2F}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VRCFaceTracking.SDK", "..\VRCFaceTracking\VRCFaceTracking.SDK\VRCFaceTracking.SDK.csproj", "{9CE3DF22-4559-4788-9680-FDF924D44D4E}"
```
Comment (rogermiranda1000) [inline PicoStreamingAssistantFTUDP.sln:18]: I don't recommend adding unrelated projects to the project .sln

**Comment by lonelyicer** [comment]
> If I were you, I'd try to parse only the file that we need (faceTrackingTransferProtocol), and ignore everything else. That way we'll avoid problems in the future, while keeping compatibility. Thanks.

I didn't think it through very well, sorry, I'll try to fix that part

**Comment by rogermiranda1000** [comment]
@lonelyicer I've pushed https://github.com/regzo2/PicoStreamingAssistantFTUDP/pull/27 that is simpler, and provides backwards compatibility.

**Comment by rogermiranda1000** [comment]
I think the other alternative https://github.com/regzo2/PicoStreamingAssistantFTUDP/pull/27 is better, and also changes less files (like the .sln project files)

**Pull request closed without merging**